src/main/scala/ai/privado/utility/SQLParser.scala

```diff
@@ -62,11 +62,8 @@ object SQLParser {
       case plainSelect: PlainSelect if plainSelect.getFromItem.isInstanceOf[Table] =>
         val sqlTable = createSQLTableItem(plainSelect.getFromItem.asInstanceOf[Table])
         List(SQLQuery(SQLQueryType.SELECT, sqlTable, getColumns(plainSelect, sqlTable)))
-      case plainSelect: PlainSelect if plainSelect.getFromItem.isInstanceOf[ParenthesedSelect] =>
-        createSQLNodesForSelect(List(plainSelect.getFromItem.asInstanceOf[ParenthesedSelect].getSelect))
-          .getOrElse(List.empty[SQLQuery])
       case parenthesedSelect: ParenthesedSelect =>
-        createSQLNodesForSelect(List(parenthesedSelect.getSelect)).getOrElse(List.empty[SQLQuery])
+        createSQLNodesForSelect(List(parenthesedSelect.getPlainSelect)).getOrElse(List.empty[SQLQuery])
```

Comment:
Does using getSelect result in some problem?

Reply:
I pulled in dev, which had this code.